### PR TITLE
8255969: Improve java/io/BufferedInputStream/LargeCopyWithMark.java using jtreg tags

### PR DESCRIPTION
**Pull request by bplb**
Please review this change which would replace the use of a system property and a child process with updated jtreg tags.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Testing

|     | Linux x64 | Linux x86 | Windows x64 | macOS x64 |
| --- | ----- | ----- | ----- | ----- |
| Build | ✔️ (5/5 passed) | ✔️ (2/2 passed) | ✔️ (2/2 passed) | ✔️ (2/2 passed) |
| Test (tier1) | ✔️ (9/9 passed) | ✔️ (9/9 passed) | ✔️ (9/9 passed) | ✔️ (9/9 passed) |

### Issue
 * [JDK-8255969](https://bugs.openjdk.java.net/browse/JDK-8255969): Improve java/io/BufferedInputStream/LargeCopyWithMark.java using jtreg tags

### Reviewers
 * [Naoto Sato](https://openjdk.java.net/census#naoto) (@naotoj - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/1083/head:pull/1083`
`$ git checkout pull/1083`

### CODE REVIEW

File: test/jdk/java/io/BufferedInputStream/LargeCopyWithMark.java [hunk 1]
```diff
@@ -23,65 +23,43 @@
 
 /* @test
  * @bug 7129312
+ * @requires (sun.arch.data.model == "64"  & os.maxMemory > 4g)
  * @summary BufferedInputStream calculates negative array size with large
  *          streams and mark
  * @library /test/lib
```
Comment (naotoj) [inline test/jdk/java/io/BufferedInputStream/LargeCopyWithMark.java:29]: Looks like this can be removed. Otherwise looks good.

Comment (bplb) [reply test/jdk/java/io/BufferedInputStream/LargeCopyWithMark.java]: Yes, you are correct: fixed. Thanks.

**Review by naotoj — APPROVED** [review]
Looks good. Thanks for fixing.

**Pull request closed without merging**